api/v1beta1/hyperconverged_types.go

```diff
@@ -798,6 +808,14 @@ type DataImportCronTemplateStatus struct {
 	Status DataImportCronStatus `json:"status,omitempty"`
 }
 
+// NodeInfoStatus holds information about the cluster nodes
+type NodeInfoStatus struct {
+	// WorkloadsArchitectures is a distinct list of the CPU architectures of the workloads nodes in the cluster.
+	WorkloadsArchitectures []string `json:"workloadsArchitectures,omitempty"`
+	// ControllerNodeArchitecture is a distinct list of the CPU architecture of the control-plane nodes.
+	ControllerNodeArchitecture []string `json:"controllerNodeArchitecture,omitempty"`
```

Comment:
"controller node" is not a well-defined term. maybe use `ControlPlaneNodesArchitecture` instead?

Reply:
Fixed